src/Common/WKB.cpp

```diff
@@ -1,6 +1,7 @@
 #include <variant>
 #include <Common/Exception.h>
 #include <Common/WKB.h>
+#include "Core/Field.h"
```

Comment:
I don't see Field being used here. If it were used, please use `<Core/Field.h>`, not `"Core/Field.h"`